src/logger_handler/logger_handler.go

```diff
@@ -54,42 +53,35 @@ func (handler *LoggerHandler) Log(loggable Loggable) {
 }
 
 func (handler *LoggerHandler) UpdateMessage(topic string, payload json.RawMessage, source string) {
-	handler.trace.Debug().Str("topic", topic).Str("source", source).Msg("update message")
+	handler.trace.Info().Str("topic", topic).Str("source", source).Msg("update message")
 	switch topic {
 	case handler.config.Topics.Enable:
-		handler.handleEnable(payload, source)
-	}
-
-	handler.notifyState()
-}
+		var enable bool
+		err := json.Unmarshal(payload, &enable)
+		if err != nil {
+			handler.trace.Error().Stack().Err(err).Msg("unmarshal enable")
```

Comment:
add a return because if not a random message can change the logger state
```suggestion
			handler.trace.Error().Stack().Err(err).Msg("unmarshal enable")
			return
```